src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -110,7 +110,7 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
             subtitleTranslationKey: 'workspace.moreFeatures.expensifyCard.subtitle',
             isActive: policy?.areExpensifyCardsEnabled ?? false,
             pendingAction: policy?.pendingFields?.areExpensifyCardsEnabled,
-            disabled: !isEmptyObject(cardsList),
+            disabled: CardUtils.shouldExpensifyCardToggleBeDisabled(policy?.workspaceAccountID, policy?.areExpensifyCardsEnabled),
```

Comment:
This makes sense as a fix, but how do we make sure the same does not happen in the cards list page when the user navigates to the page fast enough before the workspaceAccountID is populated

Reply:
In theory, it still can happen, but  I wasn't able to cause a crash with different testing scenarios:
- quickly navigating to the expensify cards page after enabling the feature
- enabling the cards feature offline -> navigating to the cards page -> go online

Anyway, the source problem will be fixed right inside the `useOnyx` hook, so I think we can leave the rare case for it.

Reply:
Sounds good to me, this should fix the main issue for us